essencium-backend/src/main/java/de/frachtwerk/essencium/backend/service/RoleService.java

```diff
@@ -45,8 +45,10 @@
 @RequiredArgsConstructor
 public class RoleService {
 
-  private static final Logger LOG = LoggerFactory.getLogger(RoleService.class);
+  public static final String ADMIN =
+      "ADMIN"; // this role will have all rights attached (see DefaultRoleInitializer)
```

Comment:
The comment is not correct. The default implementation of `DefaultRoleInitializer.getAdminRights()` grants the admin role all rights of the `BasicApplicationRights`.